### PR TITLE
Checkout api

### LINKED ISSUE #34: Add checkout api models
create models for checkout api

### PR DESCRIPTION
**Pull request by Gregsen**
fixes #34

### CODE REVIEW

File: src/models/guest_checkout_models.js [hunk 1]
```diff
@@ -70,50 +14,36 @@ const CartItem = createModel({
 * @constructor
 */
 const Cart = createModel({
-  items: { key: 'items', type: 'object', model: CartItem },
-  itemsOutOfStock: { key: 'items_out_of_stock', type: 'object', model: CartItem, optional: true },
+  items: { key: 'items', type: 'object', model: Item },
+  itemsOutOfStock: { key: 'items_out_of_stock', type: 'string', optional: true },
```
Comment (ahmed-shehata) [inline src/models/guest_checkout_models.js]: Why was this changed to string?

File: src/models/guest_checkout_models.js [hunk 2]
```diff
@@ -1,64 +1,8 @@
...
-  pickupPoint: { key: 'pickup_point', type: 'object', model: PickupPoint, optional: true }
-});
-
-/**
- * @class Class for CartItem model
- * @param {String} sku - SKU of item.
- * @param {Number} quantity - Quantity of item.
- * @constructor
- */
-const CartItem = createModel({
```
Comment (ahmed-shehata) [inline src/models/guest_checkout_models.js]: Why did you drop the CartItem model in favor of Item?

File: src/models/article_models.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
+import createModel from './base_model';
+
+/**
+ * @class Class for Brand model.
+ * @param {String} name - name of brand.
+ * @constructor
+ */
+const Brand = createModel({
```
Comment (ahmed-shehata) [inline src/models/article_models.js]: Now Brand, Price, Partner etc.. are in 2 locations?

catalog_api_models and article_models

File: src/models/recommendation_models.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
-import createModel from './base_model.js';
-import { Brand, Price, Media } from './catalog_api_models.js';
+import createModel from './base_model';
+import { Brand, Price, Media } from './article_models';
```
Comment (ahmed-shehata) [inline src/models/recommendation_models.js]: In article_models, they also need to be exported

**Comment by ahmed-shehata** [comment]
👍

**Comment by Gregsen** [comment]
:+1:
